code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java

```diff
@@ -73,4 +73,49 @@
         assertEquals(expectedSecondFromEnd, pendingSecondFromEnd);
     }
 
+    @Test public void testMergeEqualLengthLists(){
+        LinkedList actual1 = new LinkedList();
+        actual1.insert(1);
+        actual1.insert(2);
+        actual1.insert(3);
+
+        LinkedList actual2 = new LinkedList();
+        actual2.insert(5);
+        actual2.insert(9);
+        actual2.insert(4);
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{3} -> {4} -> {2} -> {9} -> {1} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
```

Comment:
These tests are ok, but really try to avoid testing another method like `merge`using the `toString()` method. The problem is that if the format of `toString` gets changed down the line, then all these tests will fail even though there is nothing wrong with them.